data/net/minecraft/world/inventory/LoomMenu.mapping

```diff
@@ -1,12 +1,24 @@
 CLASS net/minecraft/world/inventory/LoomMenu
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;Lnet/minecraft/world/inventory/ContainerLevelAccess;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+		ARG 3 resultSlotItem
```

Comment:
Wrong name (and point of 'parameters follow field names'): `resultSlotItem` -> `access`.